Core/DetectorConstants.cpp

```diff
@@ -26,6 +26,7 @@ const G4double DetectorConstants::wrappingThickness = 2 * 25.4 * pow(10, -6) * m
 const G4double DetectorConstants::wrappingShift = 1 * pow(10, -5) * m;
 G4double DetectorConstants::fScinHitMergingTime = 5.0 * ns;
 G4double DetectorConstants::fChamberCenter[3] = {0.0 * cm, 0.0 * cm, 0.0 * cm};
+const G4double DetectorConstants::muonChargeRatio = 1.2766;
```

Comment:
Is this really a detector constant?

Reply:
Its not, I've put it here temporarily. I created class PrimaryCeneratorConstants, because there are a lot of number to put there.